ci/templates/ci/identities_detail.html

```diff
@@ -241,24 +237,56 @@ <h2 class="dashhead-title">Identity Management</h2>
     </table>
   </div>
 </div>
+<div class="pagination pagination-list">
+  {% if outbound_messages.previous %}
+    <a href="{{request.site.root_page.url}}?outbound_prev={{ outbound_prev }}" class="pagination__link pagination__link--previous">
+      <span class="pagination__link--arrow">&larr;</span> Prev
+    </a>
+  {% endif %}
+  {% if outbound_messages.next %}
+    <a href="{{request.site.root_page.url}}?outbound_next={{ outbound_next }}" class="pagination__link pagination__link--next">Next
```

Comment:
@KaitCrawford thanks.

That does not seem to work, variable {{outbound_prev}} return is empty.
 And this functions as expected {{request.site.root_page.url}}?outbound_next